micromanager_gui/main_window.py

```diff
@@ -84,7 +85,7 @@ def setup_ui(self):
         uic.loadUi(self.UI_FILE, self)  # load QtDesigner .ui file
 
         # set some defaults
-        self.cfg_LineEdit.setText("demo")
+        self.cfg_LineEdit.setText("MMConfig_demo.cfg")
```

Comment:
see https://github.com/tlambert03/napari-micromanager/pull/95 for a more complete fix here